### DISCUSSION TITLE
howto see my spam?

### DISCUSSION THREAD

**Discussion by MarcS1975** [docker-mailserver/docker-mailserver#3794] [Q&A]
I am trying to see the quarantined spam emails because I need to check false positives.
Where can I see those?
Do I need to create an spamassassin-rules.cf. and amend settings there or is there an out of the box default location for spam?
I am not receiving emails from certain senders and have no visibility on what happened. Can I find them in the normal logs?

**Comment by polarathene** [comment]
Documentation on this is a little sparse, but we did recently update it:

- Old mention of quarantine location for `SA_KILL` ENV: https://docker-mailserver.github.io/docker-mailserver/v13.2/config/environment/#sa_kill
- Revised version for v13.3: https://docker-mailserver.github.io/docker-mailserver/edge/config/environment/#sa_kill

![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/6fe93a56-f074-4629-a5af-b1634eaab0f4)

The older docs on that had become a bit outdated. With the DMS defaults, your junk mail should still be stored in the junk folder, an extra copy is stored in quarantine.

---

The SpamAssassin FAQ entries had a little bit of revision as part of that update: https://docker-mailserver.github.io/docker-mailserver/edge/faq/#spamassasin

The `SA_KILL` ENV has two other relevant insights for understanding it's usage and DMS defaults, also note in the `:edge` (soon to be v13.3) docs I've made sure to better link between the related spam to junk ENV settings.

![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/479099f8-9d21-4742-9c32-708694d47646)

![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/0617862f-743d-4a14-9f8a-0bb0f094c75b)

> I am not receiving emails from certain senders and have no visibility on what happened. Can I find them in the normal logs?

You should look into Postfix logs that would relate to that sort of activity. It's unlikely they're spam as shown above the defaults are fairly permissive and should deliver any spam regardless. What is more likely is Postfix is rejecting mail from those MTAs as they don't satisfy certain expectations (_valid SPF, sender domain has an MX or A DNS record, etc_).
